src/modals/send-message-modal.js

```diff
@@ -0,0 +1,34 @@
+const {
+  ActionRowBuilder,
+  ModalBuilder,
+  TextInputBuilder,
+  TextInputStyle,
+} = require('discord.js');
+
+function createSendMessageModal(channelId, userName) {
+  const modal = new ModalBuilder()
+    .setCustomId(`send-message-modal-${channelId}-${userName}`)
+    .setTitle('Send message Request');
+
+  const titleInput = new TextInputBuilder()
+    .setCustomId('titleInput')
+    .setLabel(' Message Title')
+    .setStyle(TextInputStyle.Short)
+    .setRequired(true);
+
+  const descriptionInput = new TextInputBuilder()
+    .setCustomId('descriptionInput')
+    .setLabel('Message Description')
+    .setStyle(TextInputStyle.Paragraph)
+    .setRequired(true)
+    .setMinLength(50);
```

Comment:
Not sure if this minimum can be show in the UI so the user knows why adding less characters fails?

Reply:
The modal of discord add the minimum in the ui by default